#input: command + target

int16 sessionId

# 3 tuple state representing three action elements in each CUAction. 
# 0: success 1: fail, 2: n/a, or others. 
# e.g. if MoveAction sucess, PerceptionAction fail, GraspAction N/A
# then stateLastAction = [0, 1, 2]
int16[] stateLastAction

---

# int16 sessionId
CUAction nextAction
